core/shared/platform/android/platform_internal.h

```diff
@@ -63,19 +63,17 @@ typedef sem_t korp_sem;
 
 #define bh_socket_t int
 
+#if WASM_DISABLE_BLOCK_INSN_INTERRUPT == 0
+#define OS_ENABLE_BLOCK_INSN_INTERRUPT
+#endif
```

Comment:
The feature is only required by multi-threading, had better:
#if WASM_DISABLE_BLOCK_INSN_INTERRUPT == 0 && WASM_ENABLE_THREAD_MGR != 0